testdata/sqllogictests/rpc.slt

```diff
@@ -77,3 +78,14 @@ select * from test2;
 ----
 1
 
+# Test for #1661
+statement ok
+create table t1661 (a int, b int, c int);
+
+statement ok
+insert into t1661 select 1, 2, 3;
+
+query III
+select * from t1661;
```

Comment:
Shouldn't the query here be `select a, b, c from t1661`? Even then Not sure how this test works for #1661 🤔 

Or do I understand the issue incorrectly?

Reply:
The issue here was we using the schema of the output of the operation (which in this case is a schema with a single column named `$operation`).

To illustrate, no matter which schema I provided during the create table, it was always created with the incorrect schema:

```
> \open http://localhost:6542
Connected to Cloud deployment: unknown
> create table hello (a int, b int, c int);
Table created
> insert into hello select 1, 2, 3;
Error: Error during planning: Column count doesn't match insert query!
> insert into hello select 1;
Inserted 1 row
> select * from hello;
┌────────────┐
│ $operation │
│ ──         │
│ Utf8       │
╞════════════╡
│ 1          │
└────────────┘
```

The test case ensuring we can actually insert 3 columns and properly get 3 columns back.

Reply:
I'm sure we have other tests that would catch this, but there's currently only two files running in CI for the rpc stuff, so just wanted to add it in to catch regressions before we have all the SLTs working for rpc.

Reply:
Gotcha! Yeah, this makes sense.